Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -987,9 +1207,14 @@ def build_args_dict(args: Optional[Dict[str, str]], command_args_data: Dict[str,
                 if arg in DATE_ARGUMENTS:
                     datetime_arg = arg_to_datetime(args.get(arg))
                     if datetime_arg:
-                        args[arg] = datetime_arg.strftime('%Y-%m-%d')
+                        args[arg] = datetime_arg.strftime(DATE_ARGUMENTS.get(arg, '%Y-%m-%d'))
                 args_dict[arg] = args.get(arg)
 
+    # If some args are given, we want to add more args with default value.
+    for arg_depending_on, depended_args_dict in command_args_data.get('default_added_depended_args', {}).items():
```

Comment:
About testing build_args_dict, I meant this section that you added. Correct me if i'm wrong but I don't see it being tested.

Reply:
added